src/library/library.h

```diff
@@ -95,6 +95,7 @@ class Library: public QObject {
     static const int kDefaultRowHeightPx;
 
     void setFont(const QFont& font);
+    void importPlaylistFile(const QString& playlistFile);
```

Comment:
No implementation, no fun...
Did you forget to commit the implementation of this function?

Reply:
also, please check the failing CI jobs below

Reply:
> No implementation, no fun... Did you forget to commit the implementation of this function?

Oh!! got messed up while working on a separate PR for the issue. Will push the changes in time after checking the CI jobs.

Reply:
cool.
btw you're a volunteer contributor, no one is expecting you to deliver anything 'in time' or 'asap'
that just applies to core devs in case they messed something up.. :  )